src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressEls: ['DIV', 'LI']
```

Comment:
Rather trivial and probably a matter of personal style, but I would prefer a non- abbreviated name (`keypressElements`, `keypressNodeNames` etc).